Add compatibility Sylius 1.11

xaviermarchegay opened the pull request:
Add the compatibility with Sylius 1.11

mamazu commented:
Looks like we need to update the github configuration from the base repository. It seems that the node version has changed.

xaviermarchegay commented:
Well, the base repository also fails so I pushed the node version from the github actions of sylius/sylius.

I also changed the symfony versions in composer.json

mamazu commented:
Hm, tests are failing but I think that is because of something else. Can you try to debug them?

xaviermarchegay commented:
I fixed a number of errors (exclude some testing combinations, fixed the .env, removed a unsupported node version, etc.)

The remaining error looks like: https://gitlab.com/DMore/chrome-mink-driver/-/issues/119

I am going to try to reproduce the error on my env

mamazu commented:
If you can't reproduce them, then we will just check it again manually and then merge it without the pipeline being green.

xaviermarchegay commented:
I tried different things but I cannot seem to reproduce it...

mamazu commented:
Yeah, behat can be a pain in the butt sometimes. We'll just merge it without it being green then.

mamazu commented on tests/Application/package.json:
```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "chart.js": "^2.9.3",
-    "jquery": "^3.4.0",
```
Is this backwards compatible? Otherwise we also have to find a way to only enable it conditionally.

xaviermarchegay replied:
I have no idea why I upgraded jquery, the Sylius plugin skeleton still has ^3.4.0
Anyway, I reverted the change.
